### ISSUE TITLE
Gem release

### ISSUE THREAD

**Issue by headius** [ruby/prime#11]
This issue is a placeholder for getting a gem released.

We are currently working on jruby/jruby#5576, which has as a goal to get all default gems set up the same way as CRuby. I understand that the `prime` name on rubygems.org is not available, so there's some question about how to get a released gem. However if there's no released gem, then `prime` *can not* be a default gem library.

To understand the conundrum in JRuby: we do not populate our repository with copies of any default gems. Once a gem moves to default, we delete it from our repository and source it from the gem at build time. That can't happen for `prime` (or for `matrix` or `ostruct`) until the appropriate bits are in a gem form.

I have reverted the default gemification of `prime` and other libraries as part of work on jrubyjruby#5576. Once the gems are out there, I can flip them back.

**Comment by marcandre** [comment]
Why can't we take control of the `prime` gem? It's got 2k downloads, is 23 loc and features two methods: `Prime.prime?(n)` and `Prime.primes(start, number_of_primes)`. If we are truly worried about compatibility, we could simply copy the interface (i.e. add `Prime.primes`)

**Comment by hsbt** [comment]
There is no response from the current name holder of `prime` gem for transferring request by me.

You can confirm it.

```
$ curl https://rubygems.org/api/v1/gems/prime/owners.json
[{"id":58848,"handle":"slmnwise","email":"slmnwise@gmail.com"}]
```

I try to take over it without the approval of the current name-holder by rubygems admin.

**Comment by hsbt** [comment]
Finally, I got the namespace of `prime`

**Comment by hsbt** [comment]
@headius I released `prime-0.1.0` same as Ruby 2.6 version.

https://rubygems.org/gems/prime

ref. https://github.com/ruby/tracer/issues/1#issuecomment-610279903

**Comment by headius** [comment]
@hsbt Thank you sir!
